asapdiscovery-docking/asapdiscovery/docking/workflows/workflows.py

```diff
@@ -31,6 +31,13 @@ class WorkflowInputsBase(BaseModel):
         False, description="Whether to use the Postera database as the query set."
     )
 
+    postera_upload: bool = Field(
+        False, description="Whether to upload the results to Postera."
+    )
+    postera_molset_name: Optional[str] = Field(
+        None, description="The name of the molecule set to upload to."
+    )
+
```

Comment:
You could put these in their own inputs class (`PosteraUploadInputs` and also subclass it in large and small scale) but I'd rather not have these in the DockingWorkflowInputs bc they are specific to the large and small scale docking workflow?

Reply:
Ok sounds good, subclassing seems like a good option.